sdk/appcenter-codepush/build.gradle

```diff
@@ -0,0 +1,7 @@
+project.description = 'A cloud service that enables developers to deploy mobile app updates directly to their users` devices.'
+evaluationDependsOn(':sdk')
+
+dependencies {
+    compile project(':sdk:appcenter')
+    compile "com.google.code.gson:gson:2.8.2"
```

Comment:
Is it necessary?

Reply:
@jaelim-ms Yes, we have discussed it recently with the members of the team and decided that it's fine to use gson library. We have a lot of data classes in our project and it is more comfortable for us to use this library than serializing/de serializing manually.